pipelines/smartseq2_single_sample/GroupQCOutputs.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Aug 21 11:07:46 2018
+
+@author: jishuxu
+"""
+
+from crimson import picard
+import argparse
+import os
+import pandas as pd
+
+def AggregatePicardMetricsRow(filenames, output_name):
```

Comment:
For consistency with our other python code, please rename use underscore_case for function names